mindjunkies/accounts/tests.py

```
@@ -0,0 +1,56 @@
+from courses.models import Course, Enrollment, CourseTeacher
+
+User = get_user_model()  # Use custom User model
+
+class HomeViewTest(TestCase):
+    def setUp(self):
+        """Set up test environment"""
+        self.client = self.client  # Use Django’s test client
+        self.user = User.objects.create_user(
+            username="testuser",
+            email="testuser@example.com",  # Fix missing email
+            password="testpass"
```

Ruff (0.8.2)

Possible hardcoded password assigned to argument: "password"

(S106)